### ISSUE TITLE
stm32f4 black v2

### ISSUE THREAD

**Issue by cnc4less** [bubulindo/STM32Encoder#1]
your lib works great on stm32f1 with stm32duino maple official branch, i tried to use it with stm32f407 but could not make work , any configuration required ?

**Comment by bubulindo** [comment]
Hello,

I developed it for F1 as it's the only boards I have at the house.
Does the library compile for F4 boards?
Do you get any type of error?
Are the pins correctly configured for that board?

If you could give me a bit more info I can have a go at understanding what
may be going on.

Thanks for using my code by the way!

**Comment by cnc4less** [comment]
yes the library complies fine but it will  will generate noisy random numbers as if the encoder pins are not connected,  it is not your code i think there must be a code added to assign the right pins correctly, i wrote a simple  encoder library with attachinterrup like the basic arduino ones  and does work , again it is how to assign pins , it needs additional code  to override whatever assigned by stm32arduino on  startup.
